ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,131 @@
+
+.. _beginner-cloud:
+
+Confluent Cloud CLI
+===================
+
+Overview
+--------
+
+You can use `Confluent Cloud CLI
+<https://docs.confluent.io/current/cloud/cli/install.html#ccloud-install-cli?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
+to interact with your `Confluent Cloud
+<https://confluent.cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
+cluster.
+
+`start.sh <start.sh>`__ is a fully scripted demo that shows you how to interact
+with |ccloud|. Using the |ccloud| CLI, you can complete the following workflow
+in about 8 minutes:
+
+-  Create a new environment and specify it as the default
+-  Create a new |ak| cluster and specify it as the default
+-  Create a user key/secret pair and specify it as the default
+-  Produce and consume with |ccloud| CLI
+-  Create a service account key/secret pair
+-  Run a Java producer: before and after ACLs
+-  Run a Java producer: showcase a Prefix ACL
+-  Run |kconnect| and kafka-connect-datagen connector with permissions
+-  Run a Java consumer: showcase a Wildcard ACL
+-  Delete the API key, service account, |ak| topics, |ak| cluster, environment,
+   and the log files
+
+Prerequisites
+~~~~~~~~~~~~~~
+
+-  Access to a |ccloud| cluster
+-  Local install of `Confluent Cloud
```

Comment:
Check link -- doesn't go to CLI page

Reply:
@ybyzek The links seems to work for me. When I click on the link, it goes to the **Install the Confluent Cloud CLI** page. Is there another page that it should be redirected to?

Reply:
I will build and check

Reply:
Updated link